src/index.js

```diff
@@ -1,31 +1,68 @@
 import './style.css';
 
 const toDoContainer = document.getElementById('todolist');
+const itemInput = document.getElementById('input-item');
 
-const toDoTasks = [{
-  description: 'Sleep',
-  completed: false,
-  index: 1,
-},
-{
-  description: 'Play',
-  completed: false,
-  index: 2,
-},
-{
-  description: 'Eat',
-  completed: false,
-  index: 3,
-}];
-
-toDoTasks.sort((a, b) => a.index - b.index);
-
-toDoTasks.forEach((element) => {
-  toDoContainer.innerHTML += `
-        <ul class="list-items">
-            <li><input type="checkbox" name="item" class="check-for-items"></li>
-            <li>${element.description}</li>
-            <li class="ellipse"><i class="fa-solid fa-ellipsis-vertical"></i></li>
-        </ul>
-        `;
+function Tasks(description, index) {
+  this.description = description;
+  this.completed = false;
+  this.index = index;
+}
+
+class Books {
+  constructor() {
+    this.toDoTasks = localStorage.books ? JSON.parse(localStorage.books) : [];
+  }
+
+    updateToDoList = () => {
+      toDoContainer.innerHTML = '';
+      for (let i = 0; i < this.toDoTasks.length; i += 1) {
+        toDoContainer.innerHTML += `
+    <div class="list-items">
+        <input type="checkbox" name="item" class="check-for-items" id="${this.toDoTasks[i].index}" onchange='ticked(${this.toDoTasks[i].index})'>
+        <input type="text" value="${this.toDoTasks[i].description}" class="todo-item-text">
+        <i class='material-icons delete-btn' onclick="books.remove(${this.toDoTasks[i].index})">delete</i>
+    </div>`;
+      }
+    }
+
+    addEntry = (book) => {
+      this.toDoTasks.push(book);
+      this.saveToDoList();
+      this.updateToDoList();
+    };
+
+    saveToDoList = () => {
+      localStorage.setItem('books', JSON.stringify(this.toDoTasks));
+    };
+
+    remove(ids) {
+      this.toDoTasks = this.toDoTasks.filter((element) => element.index !== ids);
+      this.renewIndex();
+      this.saveToDoList();
+      this.updateToDoList();
+    }
+
```

Comment:
- [ ] When I try to remove a task I get the following error please make sure to fix it so that the user can remove the tasks 
<img width="911" alt="Screenshot 2023-05-11 at 19 22 00" src="https://github.com/devalibello/to-do-list/assets/61790587/7e10b6d6-da62-4f30-8040-c7ee8816d272">